src/targets/maven.ts

```diff
@@ -0,0 +1,172 @@
+import { TargetConfig } from '../schemas/project_config';
+import { MavenReleaser } from '../utils/mavenDeployment';
+import { BaseArtifactProvider } from '../artifact_providers/base';
+import { BaseTarget } from './base';
+import { ConfigurationError } from '../utils/errors';
+import { withTempDir } from '../utils/files';
+import { isDryRun } from '../utils/helpers';
+import * as Github from '@octokit/rest';
+import {
+  getAuthUsername,
+  getGithubApiToken,
+  getGithubClient,
+  GithubRemote,
+} from '../utils/githubApi';
+import simpleGit from 'simple-git';
+import { homedir } from 'os';
+import { join } from 'path';
+import * as fs from 'fs';
+
+// TODO: add docs to the readme
+
+const GIT_REPO_OWNER = 'getsentry';
+const GIT_REPO_NAME = 'sentry-java';
```

Comment:
None of this should be hard-coded, see my comment below regarding git operations.